pkg/reconciler/knativeserving/knativeserving_controller.go

```diff
@@ -196,13 +196,7 @@ func (r *ReconcileKnativeServing) install(instance *servingv1alpha1.KnativeServi
 	log.V(1).Info("install", "status", instance.Status)
 	defer r.updateStatus(instance)
 
-	extensions, err := platforms.Extend(r.client, r.kubeClientSet, r.dynamicClientSet, r.scheme)
-	if err != nil {
-		return err
-	}
-
-	err = r.executeInstall(extensions, instance)
-
+	err := r.executeInstall(instance)
```

Comment:
Personal preference: extract the transform into a separate function, and extract the status update to a separate function so the install function tells the story of
transform
applyAll
updateStatus

It would be totally fine to inline as well. But I personally prefer more small functions.

Reply:
done, thx